devtools/common/src/main/resources/templates/basic-rest/scala/pom-template.ftl

```diff
@@ -0,0 +1,171 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>${project_groupId}</groupId>
+    <artifactId>${project_artifactId}</artifactId>
+    <version>${project_version}</version>
```

Comment:
AFAICS, this file should be `pom.xml-template.ftl`.

Otherwise, when doing:
```
mvn io.quarkus:quarkus-maven-plugin:999-SNAPSHOT:create \
    -DprojectGroupId=org.acme \
    -DprojectArtifactId=rest-scala \
    -DclassName="org.acme.rest.GreetingResource" \
    -Dpath="/greeting" \
    -Dextensions="scala,resteasy-jsonb"
```

I have the following error:

> [ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:999-SNAPSHOT:create (default-cli) on project standalone-pom: Template resource is missing: /templates/basic-rest/scala/pom.xml-template.ftl -> [Help 1]

I wonder if the kotlin template is currently broken by the way as you copied the structure from the Kotlin one. We should check that, once you have fixed the extension/template detection.

Reply:
I have renamed the file for Kotlin as well.